dask_kubernetes/operator/core.py

```diff
@@ -0,0 +1,167 @@
+import subprocess
+import json
+import re
+import time
+
+from distributed.core import rpc
+from distributed.deploy import Cluster
+
+from distributed.utils import Log, Logs
+
+from dask_kubernetes.auth import ClusterAuth
+from .daskcluster import (
+    build_cluster_spec,
+    build_worker_group_spec,
+)
+
+
+class KubeCluster2(Cluster):
+    """Launch a Dask Cluster on Kubernetes using the Operator"""
+
+    def __init__(
+        self,
+        name,
+        namespace="default",
```

Comment:
I wonder if we can set this to `None` and use whatever namespace they have set in their context if they don't set it.

Reply:
And if they don't have is set in their context, should we go with 'default'? @jacobtomlinson

Reply:
We should just omit the `-n` flag from the command if this is set to `None`.